src/lib/common/Modal.svelte

```diff
@@ -6,6 +6,7 @@
 </script>
 
 <!-- svelte-ignore a11y-click-events-have-key-events -->
+<!-- svelte-ignore a11y-no-static-element-interactions -->
```

Comment:
We use the big background div as a clickable thing to close the modal. Using a div for clicking goes against a bunch of a11y guidelines, and there's a new warning here to silence.

But actually, we could probably write this component in a more modern way using [<dialog>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog). (Tip thanks to https://www.youtube.com/watch?v=ywtkJkxJsdg)